Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,86 @@
+#include <iostream> 
+#include "Person.h"
+
+
+
+
+std::string(n);
+int a;
+
+Person::Person() 
+{
+
+}
+
+Person::~Person() 
+{
+
+}
+
+Person::Person(const std::string& name, int age) 
+{
+	name_ = name;
+	age_ = new int(age);
+
+} 
+
+// getters and setters
+int Person::getAge() const 
+{ 
+
+	return *age_;
+
+}
```

Comment:
You have a lot of spaces. I think the return is also indented more than 4 spaces. Please be careful of spacing.